hypha/apply/projects/models/payment.py

```diff
@@ -26,6 +26,19 @@
 PAID = "paid"
 DECLINED = "declined"
 
+INVOICE_STATUS_BG_COLORS = {
+    "Submitted": "gray",
+    "Paid": "green",
+    "Resubmitted": "stone",
+    "Approved by Finance 2": "cyan",
+    "Approved by Finance 1": "blue",
+    "Approved by Staff": "violet",
+    "Changes Requested by Finance 2": "orange",
+    "Changes Requested by Staff": "rose",
+    "Changes Requested by Finance 1": "pink",
+    "Declined": "red",
+}
```

Comment:
Use full class names here, otherwise tailwind won't be able to create the crossponding css for those classes. See my comment in the html file. Also, put in both bg and fg color definition here.